src/main/kotlin/org/bibletranslationtools/fetcher/filesystemaccess/FileType.kt

```diff
@@ -0,0 +1,6 @@
+package org.bibletranslationtools.fetcher.filesystemaccess
+
+enum class FileType {
```

Comment:
I recommend this enum to have a property-parameter of the constructor.

Reply:
Done